.github/workflows/interop.yml

```diff
@@ -91,7 +91,7 @@ jobs:
     steps:
       - uses: actions/setup-node@v4
         with:
-          node-version: 18.14.0
+          node-version: 20.x
```

Comment:
:information_source: @gammazero this is fixes https://github.com/ipfs/kubo/actions/runs/16923143686/job/47953613651#step:12:21 – probably github changed their  images again